README.md

```diff
@@ -43,6 +46,28 @@ $ charts-syncer sync --from-date 2020-05-15
 
  > Date should be in format YYYY-MM-DD
 
+## Advanced Usage
+
+#### Sync charts and container images
+
+By default, charts-syncer only sync charts packages, but it does not sync the container images used by the chart. This
+feature can be enabled by setting the `relocateContainerImages: true` property in the config file.
+
+~~~yaml
+relocateContainerImages: true
+source:
+   ...
+target:
+   ...
+~~~
+
+In order to sync container images it is mandatory that the source charts includes a `.relok8s-images.yaml` file with information
```

Comment:
s/mandatory/required

I'd also replace instances of `charts` for `Helm Charts`